cryptus-app/components/wallet_viewer/nft_details/nft_properties/property_block/property_block.module.scss

```diff
@@ -0,0 +1,35 @@
+@use "/styles/variables" as v;
+
+.container {
+    flex: 1;
+    flex-basis: 50%;
+    width: 50%;
+    width: fit-content;
+}
+
+.block {
+    margin: 6px;
+    display: flex;
+    flex-direction: column;
+    background-color: #eef8fd;
+    align-items: center;
+    border-width: 2px;
+    border-radius: 10px;
```

Comment:
use global variable, we want to have the same border-radius everywhere, or like 2 or 3 sizes of border radius maximum